internal/engine/eval/trusty/config.go

```diff
@@ -40,14 +40,6 @@ type ecosystemConfig struct {
 	// evaluated depends on the `evaluate_score` field.
 	Score float64 `json:"score" mapstructure:"score" validate:"required"`
 
-	// EvaluateScore tells the trusty executor which score to use
-	// for evaluation. This is useful when the trusty API returns.
-	// The default is the summary score. If `score` or an empty string, the
-	// summary score is used.
-	// If `evaluate_score` is set to something else (e.g. `provenance`)
-	// then that score is used, which comes from the details field.
-	EvaluateScore string `json:"evaluate_score" mapstructure:"evaluate_score"`
```

Comment:
Oh this is probably why the "good" packages are now flagged as low scoring? Does it mean that everyone who deployed this profile with the old ruletype (before https://github.com/stacklok/minder-rules-and-profiles/pull/111) would get all their deps flagged?

Reply:
mmh interesting. Let me check why that one is getting flagged. The removal of `EvaluateScore` should not affect this one in particular as it has a high score and also a high provenance component.

Reply:
I could not reproduce it ([see here](https://github.com/puerco/demo-repo-python/pull/75)) it is weird because the profile is the same and the previous `EvaluateScore` is not used anymore. I'll keep looking